web/src/Web.App/Views/SchoolPlanningCreate/Start.cshtml

```diff
@@ -18,10 +18,10 @@
             Completing a deployment plan will take about 10 minutes.
         </p>
         <p class="govuk-body">
-            A full list of the data will be asked to input 
+            A full list of the data you will be asked to input
```

Comment:
I've fixed this in my other PR, so you'll probably need to revert this change

Reply:
reverted